src/deluge/hid/buttons.cpp

```diff
@@ -97,6 +97,16 @@ ActionResult buttonAction(deluge::hid::Button b, bool on, bool inCardRoutine) {
 		}
 	}
 
+	if (b == AFFECT_ENTIRE) {
```

Comment:
I'm not sure about making a shortcut for this, I think it would be better triggered by the menu. I imagine most people won't change often and there's a limited number of 2 button shortcuts

Reply:
Could it be a three-button shortcut? Or perhaps designate one 2 button shortcut as the "dev feature" shortcut which could be reused by multiple dev options, but only one at a time?

I like the idea of being able to verify the OLED and 7SEG state of the same sub-menu/sub-state whithout needing to lose all context by exiting the current view; messing with the settings menu system; and then hopefully recreating the exact same sub-state but completely losing the flow of what I was working on.

Reply:
That's true and I can definitely see the appeal. A 3 button shortcut seems appropriate to me!

Reply:
bikeshed proposal: shift+learn+affect-entire

Reply:
updated with this combination (shift+learn+affect-entire)